### ISSUE TITLE
fixinvT is an int but is initialised to 0.1

### ISSUE THREAD

**Issue by ianhinder** [ScottishCovidResponse/SCRCIssueTracking#462]
```
PMCMC.cc:37:22: warning: implicit conversion from 'double' to 'int' changes value from 0.1 to 0
      [-Wliteral-conversion]
        const int fixinvT = 0.1;                   // Detemines if the inverse temperature invT is fixe...
                  ~~~~~~~   ^~~
```

**Comment by chrispooley** [comment]
Thanks. Surprising the compiler doesn't give a warning.

**Comment by ianhinder** [comment]
The compiler *did* give a warning, that's how I noticed it!  I was using clang.  By the way, you don't seem to routinely compile with all warnings. If you compile with "-Wall", there are currently [135 warnings](https://projectdata.scrc.uk/coronapmcmc/branches/chrispooley/workflows/CI/latest/compile-log.html).  Some of these might indicate real problems - that was the case with BICI, where fixing some of these made the code stop crashing on macOS.

PS: Why did you close the issue?  As far as I can tell, it's not fixed.  Please can you leave issues open until they are fixed and pushed to github?  Thanks!

**Comment by chrispooley** [comment]
Hi Ian,

Thanks for the bug hunting. I'll try compiling with "-Wall". I didn't know about this flag!

Currently I'm doing some big structural changes to the model, so I won't be able to commit a new version until tomorrow. The I will pull in your changes and hope it all fits together.

Cheers,

Chris

ps Regarding the fixed issu,. I changed it in my code and clicked on fixed. Next time I'll wait until the next commit.





________________________________

**Comment by ianhinder** [comment]
Fixed in b7f81f83a2d4e559a5631d5086371ad7127896c2.
